### ISSUE TITLE
max on empty array now throws a fatal error

### ISSUE THREAD

**Issue by ymTm7KuhCnIjkZAl2x2m2** [php/doc-en#1292]
According to the documentation for `max`: 

    If an empty array is passed, then false will be returned and an E_WARNING error will be emitted.

However when I test 

    max(array())
    
 It actually throws an error. 
 
     Uncaught ValueError: max(): Argument #1 ($value) must contain at least one element in...

 https://www.php.net/manual/en/function.max.php

**Comment by cmb69** [comment]
Indeed, [throws a ValueError as of PHP 8.0.0](https://3v4l.org/K25sI).

**Comment by Girgias** [comment]
Duplicate of #2016

**Comment by ymTm7KuhCnIjkZAl2x2m2** [comment]
How is this a duplicate of a ticket reported over ten months later? Surely that one is a duplicate of this...

**Comment by Abdillah** [comment]
Because that number is prettier :smile:  and more coverage on `min` function too, I think.

**Comment by Girgias** [comment]
> Because that number is prettier smile and more coverage on `min` function too, I think.

That's the reason.
